storage/src/mmr/bitmap.rs

```diff
@@ -395,25 +372,35 @@ impl<H: CHasher, const N: usize> Bitmap<H, N> {
         self.mmr.update_leaf(hasher, leaf_pos, chunk).await
     }
 
-    /// Return the root of the Merkle tree over the bitmap.
-    ///
-    /// # Warning
-    ///
-    /// The root will not change when adding "0" bits unless a chunk boundary is crossed. If you
-    /// require a root digest that changes with every bit added, you can hash the value of
-    /// `bit_count()` into the result.
+    /// Return the root digest against which inclusion proofs can be verified.
```

Comment:
This is a nice benefit of this change too: root hash always changes with any new bits.